android/src/main/kotlin/com/situm/situm_flutter_wayfinding/SitumFlutterSDKPlugin.kt

```diff
@@ -136,7 +135,8 @@ class SitumFlutterSDKPlugin : FlutterPlugin, ActivityAware, MethodChannel.Method
                 }
             }
         }
-        SitumSdk.locationManager().requestLocationUpdates(locationRequest, locationListener!!)
+        SitumSdk.locationManager().addLocationListener(locationListener!!)
+        SitumSdk.locationManager().requestLocationUpdates(locationRequest)
```

Comment:
A esta locationRequest solo lle estamos metendo o buildingIdentifier, máis adiante collera os demáis valores por defecto non ?

Reply:
Si, pode crecer canto faga falta, só temos que modificar o método fromArguments en Extensions.kt.

Reply:
Si, eso xa é un clásico do SDK.